dashboard/src/app/workspaces/create-workspace/ready-to-go-stacks/ready-to-go-stacks.html

```diff
@@ -28,6 +28,14 @@
   </ng-form>
 </che-label-container>
 
+<!-- Ephemeral mode -->
+<che-label-container che-label-name="Ephemeral mode" che-label-description="In ephemeral mode workspace has no PVC attached." class="ephemeral-mode-toggle">
```

Comment:
It may make sense to apply Beau's suggestions on this stage https://github.com/eclipse/che/issues/15721#issuecomment-577708277